debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDatabaseSchema.java

```diff
@@ -87,7 +87,8 @@ public MySqlDatabaseSchema(MySqlConnectorConfig connectorConfig, MySqlValueConve
                         connectorConfig.customConverterRegistry(),
                         connectorConfig.getSourceInfoStructMaker().schema(),
                         connectorConfig.getFieldNamer(),
-                        false),
+                        false,
+                        connectorConfig.getEventConvertingFailureHandlingMode()),
```

Comment:
@jpechane it seems that `TableSchemaBuilder` class doesn't have `connectorConfig`.
if this field is not passed in constructor, we have to pass it in `createValueGenerator` and `create` method.
I think it would be better that passed by constructor.